src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinActivityConversionExtensions.cs

```diff
@@ -175,9 +175,17 @@ public bool ForEach(KeyValuePair<string, object> activityTag)
                 {
                     PeerServiceResolver.InspectTag(ref this, key, strVal);
 
-                    if (key == SpanAttributeConstants.StatusCodeKey && strVal == "Error")
+                    if (key == SpanAttributeConstants.StatusCodeKey)
```

Comment:
to confirm the full behaviour:
Every status except Unset is send to Zipkin as tags.
Error means, we send **additional** "error=true" tag. -- (This is not explicitly requested in spec, but is a nice addition)

Reply:
@cijothomas Correct. I'm going to open a PR into spec for adding the "error" flag on Zipkin & Jaeger when `Status=Error`. I don't know if it will get in though because of the freeze?